logical/src/main/java/org/apache/drill/common/logical/security/PlainCredentialsProvider.java

```diff
@@ -47,6 +51,18 @@ public PlainCredentialsProvider(@JsonProperty("credentials") Map<String, String>
     return credentials;
   }
 
+  @Override
+  @JsonIgnore
+  public void setCredential(String key, String value) {
```

Comment:
Please delete this method as it is not used anywhere.

Reply:
Removed.